extensions/sql-database-projects/src/test/project.test.ts

```diff
@@ -139,6 +139,7 @@ describe('Project: sqlproj content operations', function (): void {
 		projFilePath = await testUtils.createTestSqlProjFile(baselines.newProjectFileBaseline);
 		const project = await Project.openProject(projFilePath);
 
+		project.changeDSP(TargetPlatform.Sql130.toString());
 		let uri = project.getSystemDacpacUri(constants.masterDacpac);
 		let ssdtUri = project.getSystemDacpacSsdtUri(constants.masterDacpac);
 		should.equal(uri.fsPath, Uri.parse(path.join('$(NETCoreTargetsPath)', 'SystemDacpacs', '130', constants.masterDacpac)).fsPath);
```

Comment:
Probably swap Sql130 with Sql150, to test the default as Sql150. And then change DSP to 130 and verify the path.
Similarly for the other test as well.